app/components/LoginForm.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+  color: white;
+  background: none;
+  border: none;
+  margin: 0;
+  padding: 0;
+  cursor: pointer;
+  text-decoration: underline;
+  font-size: 0.8em;
+`;
+
+const StyledForm = styled.form`
+  margin: 0 0 0 10px;
+`;
```

Comment:
Noticed that the login button isn't aligned with the dashboard link. Adding this to the `StyledForm` css will center it properly:

` display: flex;
  align-items: center;
  justify-content: center;
`

Reply:
@BryanGK actually might not need the `justify-content` line, just the other two.